ecs-cli/modules/cli/compose/entity/service/service.go

```diff
@@ -467,6 +474,11 @@ func (s *Service) buildCreateServiceInput(serviceName, taskDefName string) (*ecs
 		Role:                    aws.String(s.role),
 	}
 
+	if schedulingStrategy == "DAEMON" {
+		createServiceInput.DesiredCount = nil
+		createServiceInput.SchedulingStrategy = aws.String(schedulingStrategy)
```

Comment:
Instead of setting this only if `DAEMON`, can we set it outside this check on the condition `schedulingStrategy != ""` ? That way any value explicitly set by the customer will be passed along and we can leave type validation to ECS. (However `DesiredCount` can remain within this `if` block since it's specific to that strategy)

Reply:
This should be also addressed with https://github.com/aws/amazon-ecs-cli/pull/685/commits/a78de9eca550319e3bb5962c4abb777bc4401b0b.

Reply:
https://github.com/aws/amazon-ecs-cli/pull/685/commits/a4b9bb03e4f8ad2ad17c48af082775e413713b99 adds this and a test which uses lowercase.